src/server/server.ts

```diff
@@ -979,6 +979,12 @@ namespace ts.server {
         allowLocalPluginLoads
     };
 
+    if (logger.hasLevel(LogLevel.normal)) {

```

Comment:
why not do it without checking level.. I think this information is good to have in any logging mode.